### DISCUSSION TITLE
Has anyone tried deploying whisper model on the NVIDIA Triton Inference Server?

### DISCUSSION THREAD

**Discussion by Kianit-bit** [triton-inference-server/server#6574] [General]
Can I run whisper on Triton?
Any link to example code will be very helpful.

**Comment by mhbassel** [comment]
In case you're interested, there's a related discussion in OpenAI github: https://github.com/openai/whisper/discussions/1505
But still no one shared any code examples.

**Comment by saitejamalyala** [comment]
https://github.com/triton-inference-server/server/issues/7847

Above Has a example using python backend (there are better ways to do it as well). But only http endpoint works unfortunately.

**Reply by dineshveguru** [reply]
Hi @saitejamalyala, Could you please tell me how to use the API. I hosted it and running fine. But, getting errors while using it

**Reply by saitejamalyala** [reply]
@dineshveguru , Can you share a gist of how you are trying to use it ?

**Reply by dineshveguru** [reply]
My intended use case is to send audio file url to the API, so that I can download and store the audio file locally in the server and then I can process it and then send the transcription through the API. 
This is how I tried to use it.
```
{
    "INPUT": "http://audiofile.wav"
}
```
Thank you.

**Reply by saitejamalyala** [reply]
hf pipeline has the capability to use the url as the source, do you have more information regarding the error ?
